timemachine/fe/free_energy.py

```diff
@@ -342,7 +342,7 @@ def _combine(self, ligand_values_a, ligand_values_b, host_values=None):
         -------
             combined_values
         """
-        if isinstance(self.top, topology.SingleTopology):
+        if isinstance(self.top, single_topology.SingleTopology):
             ligand_values = [np.mean(self.top.interpolate_params(ligand_values_a, ligand_values_b), axis=0)]
```

Comment:
Can this branch be removed? According to the constructor annotation, `self.top` should be `DualTopology`.

Reply:
good call, willfix.